From 3c4834895a0eb793ba1aac219b768997aecccfb6 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Wed, 8 Feb 2017 17:23:43 +0800
Subject: [PATCH 1500/5242] MLK-13895 ARM: dts: imx7ulp: correct the device
 node name of lpi2c5

commit  3feafd280fb917d7fb3e6a0dd3baf896b02466be from
https://source.codeaurora.org/external/imx/linux-imx.git

Correct this spelling mistake error for lpi2c5 node.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7ulp.dtsi b/arch/arm/boot/dts/imx7ulp.dtsi
index f36c908..6ec834b 100644
--- a/arch/arm/boot/dts/imx7ulp.dtsi
+++ b/arch/arm/boot/dts/imx7ulp.dtsi
@@ -208,7 +208,7 @@
 			assigned-clock-rates = <48000000>;
 		};
 
-		lpi2c5: lpi2c4@402C0000 {
+		lpi2c5: lpi2c5@402C0000 {
 			compatible = "fsl,imx7ulp-lpi2c";
 			reg = <0x402C0000 0x10000>;
 			interrupts = <GIC_SPI 35 IRQ_TYPE_LEVEL_HIGH>;
-- 
1.7.9.5

